src/components/select/select.scss

```diff
@@ -25,8 +25,9 @@
     
     &__value-arrow {
         position: absolute;
-        top: 8px;
+        top: 50%;
         right: 0;
+        transform: translateY(-50%);
```

Comment:
Confirmed this is [supported on all modern browsers](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/translateY#browser_compatibility) (I believe this didn't work on some older browsers back in the day so wanted to double-check)

Reply:
Also could this be simplified to `top: 37.5%`?

Reply:
The answer is no, see [below comments](https://github.com/argoproj/argo-ui/pull/178#issuecomment-2157301364)